pkg/k8s/volumes/translate.go

```diff
@@ -33,10 +33,20 @@ func translate(name string) *apiv1.PersistentVolumeClaim {
 
 //GetVolumeName returns the okteto volume name for a given dev environment
 func GetVolumeName(dev *model.Dev) string {
-	return fmt.Sprintf(oktetoVolumeTemplate, dev.Name)
+	n := fmt.Sprintf(oktetoVolumeTemplate, dev.Name)
+	if len(n) > 63 {
+		n = n[0:63]
+	}
+
+	return n
 }
 
 //GetVolumeDataName returns the okteto volume name for a given dev environment
 func GetVolumeDataName(dev *model.Dev, i int) string {
-	return fmt.Sprintf(oktetoVolumeDataTemplate, dev.Name, i)
+	n := fmt.Sprintf(oktetoVolumeDataTemplate, dev.Name, i)
```

Comment:
I wouldn´t have the index at the end if we are trimming the volume name

Reply:
ok, would changing this break existing deployments?

Reply:
no. it would recreate the volume, but this feature is not documented yet